libxml2/PKGBUILD

```diff
@@ -1,42 +1,68 @@
 # Maintainer: Alexey Pavlov <alexpux@gmail.com>
 
 pkgname=('libxml2' 'libxml2-devel' 'libxml2-python')
-pkgver=2.9.2
-pkgrel=3
+pkgver=2.9.5
+pkgrel=1
 pkgdesc="XML parsing library, version 2"
 arch=(i686 x86_64)
 license=('MIT')
-makedepends=('python2' 'python3' 'libreadline-devel' 'ncurses-devel' 'liblzma-devel' 'zlib-devel')
+makedepends=('python2' 'python3' 'icu-devel>=59.1' 'libreadline-devel' 'ncurses-devel' 'liblzma-devel' 'zlib-devel')
 url="http://www.xmlsoft.org/"
 source=(http://xmlsoft.org/sources/${pkgname}-${pkgver}.tar.gz
         https://www.w3.org/XML/Test/xmlts20130923.tar.gz
         libxml2-2.9.0-do-not-check-crc.patch
         libxml2-2.7.3-doc-install.patch
         libxml2-2.8.0_rc1-winnt.patch
-        libxml2-2.9.1-msys2.patch
-        libxml2-2.9.2-catalog-revert.patch)
-sha256sums=('5178c30b151d044aefb1b08bf54c3003a0ac55c59c866763997529d60770d5bc'
+        libxml2-2.9.1-msys2.patch)
+sha256sums=('4031c1ecee9ce7ba4f313e91ef6284164885cdb69937a123f6a83bb6a72dcd38'
             '9b61db9f5dbffa545f4b8d78422167083a8568c59bd1129f94138f936cf6fc1f'
             '5a53d640ebce5a96e35fbdf5f644c37642dfa3345a1dc4fc355728897c19b7e7'
             'bff6790eadec6d7b3affd2d3a7385ce7f80cec2cf4abbd2ed83e9cb297a614a9'
             'c18b2c15550b3ff4709a85cd18d16368928319b1b7fd8875dc34acd67076b134'
-            '3eadad15d01594b8b52e561a4ebce3dffe16a42e5ebd211ea50df92a244994ed'
-            'ea2bcb58b4b39835c031aa64433c01f885533a33a1e64069633c43dc4ef8be31')
+            '3eadad15d01594b8b52e561a4ebce3dffe16a42e5ebd211ea50df92a244994ed')
 
 PYTHON_SITELIB=/usr/lib/python2.7/site-packages
 PYTHON_LIB=/usr/lib/python2.7
 
+# Helper macros to help make tasks easier #
+apply_patch_with_msg() {
+  for _patch in "$@"
+  do
+    msg2 "Applying $_patch"
+    patch -Nbp1 -i "${srcdir}/$_patch"
+  done
+}
+
+apply_patch_p2_with_msg() {
+  for _patch in "$@"
+  do
+    msg2 "Applying $_patch"
+    patch -Nbp2 -i "${srcdir}/$_patch"
+  done
+}
+
+del_file_exists() {
+  for _fname in "$@"
```

Comment:
What's wrong with calling `rm -rf ` or `rm -f` always?

Reply:
Sometimes, that might trigger an error if the file or dir doesn't exist.

Reply:
That only happens without the `-f` flag.